packages/generator/templates/operations.ts.hbs

```diff
@@ -2,10 +2,13 @@
   {{#each operations}}
     /**
     * {{description}}
+    * 
+    * If you would like to get a raw Response object use the other {{name}} function.
+    * 
+    * @returns A promise of type {{getReturnPayloadType .}}
```

Comment:
Should this be Response | {{getReturnPayloadType .}}

Reply:
This overload does not return Response. It's the one below this one that returns it. tsdoc for that one captures that.